### PR TITLE
update references for osm to v0.9

### PR DESCRIPTION
**Pull request by zr-msft**
relates to #108
relates to #106

### CODE REVIEW

**Comment by zr-msft** [comment]
@shashankram

can you review my changes here: https://github.com/openservicemesh/osm-docs/pull/109/files?file-filters%5B%5D=.md#diff-a0083d990846bc272444620502a478629a1376697b518ef046dfd3858e848296

the rendered preview makes the context a bit clearer:
https://deploy-preview-109--osm-docs.netlify.app/docs/tasks_usage/certificates/?utm_source=github&utm_campaign=bot_dp#types-of-certificates

File: content/docs/control_plane_health_probes.md [hunk 1]
```diff
@@ -123,8 +123,8 @@ If any health probes are consistently failing, perform the following steps to id
     $ osm mesh list
 
     MESH NAME   NAMESPACE      CONTROLLER PODS                  VERSION     SMI SUPPORTED
-    osm         osm-system     osm-controller-5494bcffb6-qpjdv  v0.8.4      TrafficSplit:split.smi-spec.io/v1alpha2,TrafficTarget:access.smi-spec.io/v1alpha3,HTTPRouteGroup:specs.smi-spec.io/v1alpha4,TCPRoute:specs.smi-spec.io/v1alpha4
-    osm2        osm-system-2   osm-controller-48fd3c810d-sornc  v0.8.4      TrafficSplit:split.smi-spec.io/v1alpha2,TrafficTarget:access.smi-spec.io/v1alpha3,HTTPRouteGroup:specs.smi-spec.io/v1alpha4,TCPRoute:specs.smi-spec.io/v1alpha4
+    osm         osm-system     osm-controller-5494bcffb6-qpjdv  v0.9.0      TrafficSplit:split.smi-spec.io/v1alpha2,TrafficTarget:access.smi-spec.io/v1alpha3,HTTPRouteGroup:specs.smi-spec.io/v1alpha4,TCPRoute:specs.smi-spec.io/v1alpha4
+    osm2        osm-system-2   osm-controller-48fd3c810d-sornc  v0.9.0      TrafficSplit:split.smi-spec.io/v1alpha2,TrafficTarget:access.smi-spec.io/v1alpha3,HTTPRouteGroup:specs.smi-spec.io/v1alpha4,TCPRoute:specs.smi-spec.io/v1alpha4
```
Comment (bridgetkromhout) [inline content/docs/control_plane_health_probes.md:127]: Might be a good idea to check and see if the SMI SUPPORTED items have any particular sort order and if so, did the default change; I tested with the v0.9.0 RC and got a different order (HTTPRouteGroup was first).

File: content/_index.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ OSM runs an Envoy based control plane on Kubernetes, can be configured with SMI
 ## Install
 
 ### Prerequisites
-- Kubernetes cluster running Kubernetes v1.18.0 or greater
+- Kubernetes cluster running Kubernetes v1.19.0 or greater
```
Comment (bridgetkromhout) [inline content/_index.md:29]: Are you basing this on what's supported in terms of k8s versioning? Should OSM have a version skew policy published somewhere that we then link here? https://github.com/openservicemesh/osm#prerequisites still says 1.18, so we shouldn't change it without dev team buy-in.

File: content/docs/install/_index.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ weight: 3
 
 ## Prerequisites
 
-- Kubernetes cluster running Kubernetes v1.18.0 or greater
+- Kubernetes cluster running Kubernetes v1.19.0 or greater
```
Comment (bridgetkromhout) [inline content/docs/install/_index.md:11]: Same comment as above.

File: content/docs/control_plane_health_probes.md (hunk 1 above)
Comment (zr-msft) [reply content/docs/control_plane_health_probes.md]: thank you @bridgetkromhout 

fixed

File: config.toml [hunk 4]
```diff
@@ -85,8 +85,8 @@ project_home = "https://openservicemesh.io/"
   version = "latest"
   url = "https://docs.openservicemesh.io/"
 [[params.versions]]
-  version = "v0.8"
-  url = "https://release-v0-8.docs.openservicemesh.io/"
+  version = "v0.9"
+  url = "https://release-v0-9.docs.openservicemesh.io/"
```
Comment (bridgetkromhout) [inline config.toml:89]: Ah, we want to keep both (perhaps in reverse order?) as there will be customers who want the v0.8.x docs for some time. So when you add v.09, also keep a v.08 one too. You can have an arbitrary number of `version = "whatever` choices, each with a url.

File: content/_index.md (hunk 2 above)
Comment (zr-msft) [reply content/_index.md]: moved to #111

File: content/docs/install/_index.md (hunk 3 above)
Comment (zr-msft) [reply content/docs/install/_index.md]: moved to #111

File: config.toml (hunk 4 above)
Comment (zr-msft) [reply config.toml:89]: thank you @bridgetkromhout  for the feedback.

fixec

File: content/docs/tasks_usage/traffic_management/iptables_redirection.md [hunk 5]
```diff
@@ -217,7 +217,7 @@ The following snippet from the demo `curl` client's init container spec shows th
 Init Containers:
   osm-init:
     Container ID:  containerd://80f86af7bc64b7a70f7f2bf64242d735d857559a79cd97e206513368130902f1
-    Image:         openservicemesh/init:v0.8.0
+    Image:         openservicemesh/init:v0.9.0
```
Comment (nojnhuh) [inline content/docs/tasks_usage/traffic_management/iptables_redirection.md:220]: This is probably also a spot we don't need to update for every release. No issue with including it here though.

**Review by nojnhuh — DISMISSED** [review]
Thanks, @zr-msft! I did my own `grep` and these changes look complete to me. I left a couple comments about things that I don't think necessarily need to be updated with each release, but certainly doesn't hurt to include them here.

/cc @allenlsy Let's update your list in https://github.com/openservicemesh/osm/pull/3533 to include these files.

File: content/docs/application_health_probes.md [hunk 6]
```diff
@@ -328,13 +328,13 @@ Events:
   Type     Reason     Age              From               Message
   ----     ------     ----             ----               -------
   Normal   Scheduled  17s              default-scheduler  Successfully assigned bookstore/bookstore-v1-699c79b9dc-5g8zn to osm-control-plane
-  Normal   Pulled     16s              kubelet            Successfully pulled image "openservicemesh/init:v0.8.4" in 26.5835ms
+  Normal   Pulled     16s              kubelet            Successfully pulled image "openservicemesh/init:v0.9.0" in 26.5835ms
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/docs/application_health_probes.md:331]: thank you @nojnhuh

File: content/docs/control_plane_health_probes.md [hunk 7]
```diff
@@ -40,8 +40,8 @@ Events:
   Type     Reason     Age               From               Message
   ----     ------     ----              ----               -------
   Normal   Scheduled  24s               default-scheduler  Successfully assigned osm-system/osm-controller-85fcb445b-fpv8l to osm-control-plane
-  Normal   Pulling    23s               kubelet            Pulling image "openservicemesh/osm-controller:v0.8.0"
-  Normal   Pulled     23s               kubelet            Successfully pulled image "openservicemesh/osm-controller:v0.8.0" in 562.2444ms
+  Normal   Pulling    23s               kubelet            Pulling image "openservicemesh/osm-controller:v0.9.0"
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/docs/control_plane_health_probes.md:43]: thank you @nojnhuh

File: content/docs/tasks_usage/traffic_management/iptables_redirection.md (hunk 5 above)
Comment (zr-msft) [reply content/docs/tasks_usage/traffic_management/iptables_redirection.md:220]: thank you @nojnhuh

File: config.toml (hunk 4 above)
Comment (bridgetkromhout) [reply config.toml:89]: Oops, just realized - can you remove this change for now, as we need to ensure the docs are published and _then_ link to them in a second PR?

**Review by bridgetkromhout — APPROVED** [review]
LGTM - thanks.

**Pull request merged**